File: csharp/Carballo/Com/Alonsoruibal/Chess/Book/Book.cs
/*
 * Code converted with Sharpen
 * 
 */using Com.Alonsoruibal.Chess;
using Sharpen;

namespace Com.Alonsoruibal.Chess.Book
{
	/// <summary>Opening book support</summary>
	/// <author>rui</author>
	public interface Book
	{
		/// <summary>Gets a random move from the book taking care of weights</summary>
		int GetMove(Board board);
	}
}

File: core/src/main/java/com/alonsoruibal/chess/book/Book.java
package com.alonsoruibal.chess.book;

import com.alonsoruibal.chess.Board;


/**
 * Opening book support
 *
 * @author rui
 */
public interface Book {
	/**
	 * Gets a random move from the book taking care of weights
	 */
	int getMove(Board board);
}
